### ISSUE TITLE
fix output if more ns than one

### ISSUE THREAD

**Issue by gionniboy** [gionniboy/howisresolved#5]
example

```
 pipenv run python howisresolved.py --domain github.com --expect 192.30.253.112 192.30.253.113
2018-04-30 22:01:51,647 - 13641 - howisresolved.py:54 - __main__ - INFO - LOGGING SETUP from JSON logging.json
2018-04-30 22:01:51,647 - 13641 - howisresolved.py:55 - __main__ - DEBUG - LOGGING OK - path logging.json - level 20
usage: howisresolved.py [-h] [--domain DOMAIN] [--dnsfile DNSFILE]
                        [--dnsrand DNSRAND] --expect EXPECT
howisresolved.py: error: unrecognized arguments: 192.30.253.113


```

make possible multiple entry to check.

**Comment by gionniboy** [comment]
add option for RR records

**Comment by gionniboy** [comment]
closed with #9
